_component_design/tooltip.md

```diff
@@ -13,28 +13,292 @@ last-modified: 2017-08-17
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Meet the tooltip pattern"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-hero.svg){:width="100%"}
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+### When not to use
+{: .hxSubSectionTitle}
+Do not use a tooltip if users must repeatedly refer to a tooltip to understand an option or to complete a workflow. We recommend you do not use a tooltip when users needs to interact with the tip's content, or when users integrate with the site on a mobile device.
+
+### Best practice
+{: .hxSubSectionTitle}
+Tooltips are perfect for short sentences and sentence fragments that assist the user in their understanding. The ideal tooltip is not obvious, nor does it repeat content that is already visible on the site. Place a tooltip in a subtle location, and ensure it does not interfere with the user or obscure any objects of interest. Use the tooltip on icon  images, text, and table rows. The maximum width for the tooltip is 400px, ensure there are no interactive controls inside the tooltip.
+
+
+### Problems being solved
+{: .hxSubSectionTitle}
+Tooltips are a powerful way to simplify a user interface. They provide information users need when they need it, with minimal effort on their part. Tips can help you use screen space more effectively and reduce screen clutter.
+
+{% endcolumn %}
+
+</div>
+
+## Default composition 
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+Tooltips are composed of two primary portions, the tooltip container and the trigger:
+
+- The trigger is the element that displays the tooltip component when interacted with from a user. (see trigger types section)
+
+- The container houses the border, pointer, and content. 
+
+The tooltip will stand out from the background by using a lighter background colour than the interface background, a rule or line around the tooltip box, and a shadow around the tooltip so it appears raised from the items below.
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Tooltip composition."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/help/tooltips/tooltip-composition.svg){:width="100%"}
+
+{% endfigure %}
+{% endcolumn %}
+</div>
+
+## Tooltip specifications
+
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Spacing
+{: .hxSubSectionTitle}
+Tooltips have varying widths based on the amount of content, however, a tooltip relies on the following consistent spacing specifications:
+
+- 8px between vertical elements
```

Comment:
space between the number and the unit

Reply:
I want to note this is a point of contention between people right now. In the redlines we are not using spaces and in earlier docs alex said lets pick one and be consistent, with a preference towards no space. I tend to also prefer no space. I'll move forward with adding the spaces across this doc and then we can discuss this week and go back and edit the necessary docs to fit whatever the chosen path forward is. 

@asettle @araiford See above and weigh in.